guillotina/db/interfaces.py

```diff
@@ -328,6 +328,7 @@ async def terminate(conn):
 
 class IPostgresStorage(IStorage):
     objects_table_name = Attribute("")
+    blobs_table_name = Attribute("")
```

Comment:
Is this related to this PR?

Reply:
ahh, nah, I was in the mode of adding some mypy for some code. I can remove it.